src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,50 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_LENGTH = 10;
+    private Object[] storageKeys;
+    private Object[] storageValues;
+    private int lengthCounter;
+
+    public StorageImpl() {
+        storageKeys = new Object[MAX_STORAGE_LENGTH];
+        storageValues = new Object[MAX_STORAGE_LENGTH];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < lengthCounter; i++) {
+            if (compare(storageKeys[i], key)) {
+                storageValues[i] = value;
+                return;
+            }
+        }
+        storageKeys[lengthCounter] = key;
+        storageValues[lengthCounter] = value;
+        lengthCounter++;
     }
 
     @Override
     public V get(K key) {
-        return null;
+        int keyIndex = -1;
+        for (int i = 0; i < lengthCounter; i++) {
+            if (compare(storageKeys[i], key)) {
+                keyIndex = i;
+            }
+        }
+        if (keyIndex >= 0) {
+            return (V) storageValues[keyIndex];
+        } else {
+            return null;
+        }
```

Comment:
I'd recommend to simplify this way:
```suggestion
        for (int i = 0; i < lengthCounter; i++) {
            if (compare(storageKeys[i], key)) {
                return (V) storageValues[keyIndex];
            }
        }
        return null;
```